CODEOWNERS

```diff
@@ -178,7 +178,7 @@
 /src/content/docs/aws/services/pinpoint.mdx @viren-nadkarni @quetzalliwrites @HarshCasper
 
```

Comment:
@carole-lavillonniere as you can see, the convention is we add new engineers to the services they co-own like you see here... For now, you're on pipes, but as you likely get added to more services eventually, now you know how to create a PR and request the appropriate code ownership access. 🔥